### ISSUE TITLE
Check for class name case-mismatch

### ISSUE THREAD

**Issue by tomsommer** [kalessil/phpinspectionsea#1676]
Related to https://youtrack.jetbrains.com/issue/WI-38140

```
class Foo {

}

$foo = new foo;

```
Should trigger a warning, as class names do not match in a case-sensitive manner.

**Comment by ea-inspections-team** [comment]
Thanks for reporting @tomsommer !
In the linked issue, folks recommended enabling 'PHP | Code style | Case mismatch in method call or class usage', hence closing the issue.
